rocks/horizon/rockcraft.yaml

```diff
@@ -62,6 +63,26 @@ parts:
       _1372_project_container_infra_cluster_templates_panel.py
       _2370_admin_container_infra_panel_group.py
       _2371_admin_container_infra_quotas_panel.py" > $CRAFT_STAGE/$AVAILABLE/magnum
+      echo "_80_manila_admin_add_share_panel_group.py
```

Comment:
If Manila is disabled, are the UI components still enabled?

If not, then this is not necessary.

If that's the case it is.

Reply:
Do you mean all of them, or just the `_80_manila_` files?

If it's the `_80_manila_` files, I'd recommend they should stay here, it would make the UI clearer. Basically, without these, all the Shares and Shares-related tabs would be grouped up under "Other":

<img width="959" height="262" alt="snip-shares-other" src="https://github.com/user-attachments/assets/9cd050b0-33e3-419f-9acd-fc633474d8fd" />

While, with these 2 files, they would be grouped up under "Shares":

<img width="1917" height="703" alt="snip-shares" src="https://github.com/user-attachments/assets/fe644ed3-125e-4c71-b119-4b9bdadd1c47" />

Reply:
Oh, I see what you mean now. Same as the other plugins: if manila is not deployed, the manila-related UI does not show up, and horizon works with what it has.